### PR TITLE
v0.21: Update configuration options

### PR DESCRIPTION
**Pull request by guimachiavelli**
Log level
- [x] /reference/features/configuration.md: add option description (alphabetic order)

Analytics
- [x] /reference/features/configuration.html#disable-analytics
- [x] /reference/features/configuration: remove --no-sentry L#165-174
- [x] /reference/features/configuration: add line explaining sentry is now handled by --no-analytics L#163
- [x] /reference/features/configuration: link to telemetry page L#163

MAX_MDB_SIZE
- [x] /reference/features/configuration L#68, L#218, L#219, L#226, L#228
- [x] /learn/getting_started/installation L#102
- [x] /reference/features/known_limitations L#15
- [x] /reference/features/configuration: add line explaining users can specify human-readable units

HTTP-PAYLOAD-SIZE-LIMIT
- [x] /learn/core_concepts/documents: remove "must be in bytes" L#161
- [x] /reference/features/configuration: update description clarifying a user can specify the units L#263

### CODE REVIEW

File: reference/features/configuration.md [hunk 1]
```diff
@@ -193,21 +185,36 @@ MeiliSearch will only launch once the dump data has been fully indexed. The time
...
-**CLI option**: `--max-mdb-size`
+**Environment variable**: `MEILI_MAX_INDEX_SIZE`
+**CLI option**: `--max-index-size`
 **Default value**: `107374182400` (100 GiB)
 
-Sets the maximum size of the `main` database. Value must be given in bytes.
+Sets the maximum size of the index. Value must be given in bytes or explicitly stating a base unit. For example, the default value can be written as `107374182400`, `'107.7Gb'`, or `'107374 Mb'`.
 
-The `main` database stores processed data and is different from the `update` database, which handles [pending updates](/learn/advanced/asynchronous_updates.md).
+The index stores processed data and is different from the `update` database, which handles [pending updates](/learn/advanced/asynchronous_updates.md).
```
Comment (CaroFG) [inline reference/features/configuration.md:211]:
```suggestion
The `index` stores processed data and is different from the `update` database, which handles [pending updates](/learn/advanced/asynchronous_updates.md).
```

File: reference/features/configuration.md [hunk 2]
```diff
@@ -221,9 +228,9 @@ getconf PAGE_SIZE
 **CLI option**: `--max-udb-size`
 **Default value**: `107374182400` (100 GiB)
 
-Sets the maximum size of the `update` database. Value must be given in bytes.
+Sets the maximum size of the `update` database. Value must be given in bytes or explicitly stating a base unit. For example, the default value can be written as `107374182400`, `'107.7Gb'`, or `'107374 Mb'`.
 
-The `update` database handles [pending updates](/learn/advanced/asynchronous_updates.md). This is different from the `main` database, which only stores processed data.
+The `update` database handles [pending updates](/learn/advanced/asynchronous_updates.md). This is different from the index database, which only stores processed data.
```
Comment (CaroFG) [inline reference/features/configuration.md:233]:
```suggestion
The `update` database handles [pending updates](/learn/advanced/asynchronous_updates.md). This is different from the `index` database, which only stores processed data.
```

File: reference/features/configuration.md [hunk 3]
```diff
@@ -243,7 +250,7 @@ getconf PAGE_SIZE
 **CLI option**: `--http-payload-size-limit`
 **Default value**: `104857600` (~100MB)
 
-Sets the maximum size of accepted JSON payloads. Value must be given in bytes.
+Sets the maximum size of accepted JSON payloads. Value must be given in bytes or explicitly stating a base unit.
```
Comment (CaroFG) [inline reference/features/configuration.md:253]:
```suggestion
Sets the maximum size of accepted JSON payloads. Value must be given in bytes or explicitly stating a base unit.
```
Maybe also add here an example of the default value written in different units, as in the max-index-size and max-udb-size sections

**Review by CaroFG — COMMENTED** [review]
I made some suggestions, tell me if you agree or not. I'll approve whatever you decide

File: reference/features/configuration.md (hunk 1 above)
Comment (guimachiavelli) [reply reference/features/configuration.md:211]: Good one, thanks!

File: reference/features/configuration.md (hunk 3 above)
Comment (guimachiavelli) [reply reference/features/configuration.md:253]:
```suggestion
Sets the maximum size of accepted JSON payloads. Value must be given in bytes or explicitly stating a base unit. For example, the default value can be written as `107374182400`, `'107.7Gb'`, or `'107374 Mb'`.
```

**Review by CaroFG — APPROVED** [review]

**Review by curquiza — COMMENTED** [review]
Sorry for the delay, I started the review yesterday and I forgot to submit the review :(

File: learn/getting_started/installation.md [hunk 4]
```diff
@@ -99,7 +99,7 @@ To install MeiliSearch on Windows, use Docker or compile from the source.
 
 A common compilation error (`"link.exe not found"`) can be solved by installing [Visual Studio Build Tools](https://visualstudio.microsoft.com/downloads/) (scroll down and click on **Tools for Visual Studio 2019**).
 
-MeiliSearch will allocate 100GB on launch. If disk space is an issue on your machine, adjust the [main database](/reference/features/configuration.md#max-mdb-size) and [update database](/reference/features/configuration.md#max-udb-size) maximums accordingly.
+MeiliSearch will allocate 100GB on launch. If disk space is an issue on your machine, adjust the [index](/reference/features/configuration.md#max-index-size) and [update database](/reference/features/configuration.md#max-udb-size) maximums accordingly.
```
[the remark this answers is not in the document]
Comment (guimachiavelli) [reply learn/getting_started/installation.md:102]: Do you mean that changing disk space allocation is no longer necessary/possible on windows?

[the remark this answers is not in the document]
Comment (curquiza) [reply learn/getting_started/installation.md:102]: What is available for windows users now:
- the `https://install.meilisearch.com` script will not work. An issue is opened -> https://github.com/meilisearch/MeiliSearch/issues/1570
- the binary is expected to work
- there is no compilation issue anymore
- there is no issue with memory allocation anymore

[the remark this answers is not in the document]
Comment (guimachiavelli) [reply learn/getting_started/installation.md]: As we discussed, I am removing the references to special behaviour on Windows regarding the max index size. We will review the rest of the docs regarding MeiliSearch on Windows in another PR.

**Pull request merged**